docs/usage-guide.md

```diff
@@ -0,0 +1,85 @@
+# Usage Guide for Osmo-Notify
+
+## Introduction
+
+Welcome to the usage guide for Osmo-Notify, a powerful notification management system designed to simplify the process of sending notifications through various channels. This guide will walk you through the steps of integrating Osmo-Notify into your application and making the most of its features.
+
+## Table of Contents
+
+1. Overview
+2. Pushing Data to the Database
+3. Using the Osmo-Notify API
+4. Tracking Notification Status
+5. Available Channel Types
+6. Delivery Status Information
+
+## 1. Overview
+
+Osmo-Notify offers a streamlined solution for sending notifications via different channels. It operates by monitoring the `notify_notifications` database table and automatically dispatches notifications for records with a `PENDING` status.
+
+## 2. Pushing Data to the Database
+
+Developers have the flexibility to populate the `notify_notifications` database table directly or opt for the recommended approach of utilizing the Osmo-Notify API. By following this method, the notification processing is seamlessly managed by Osmo-Notify.
+
+## 3. Using the Osmo-Notify API
+
+To use the Osmo-Notify API, follow these steps:
+
+- **Method:** POST
+- **Endpoint:** `/notifications`
+
+**Sample Request Body:**
+```json
+{
+  "channelType": 1,
```

Comment:
is channel type a enum now? I think we are sending text here.

Reply:
Channel type is enum(using tinyint)